frontend/src/modules/_shared/LayerFramework/settings/settingsDefinitions.ts

```diff
@@ -0,0 +1,320 @@
+import type { SurfaceStatisticFunction_api, WellboreHeader_api } from "@api";
+import type { RegularEnsembleIdent } from "@framework/RegularEnsembleIdent";
+import type { ColorScaleConfig } from "@framework/components/ColorScaleSelector/colorScaleSelector";
+import type { ColorSet } from "@lib/utils/ColorSet";
+
+import { isEqual } from "lodash";
+
+import type { IntersectionSettingValue } from "./implementations/IntersectionSetting";
+import type { SensitivityNameCasePair } from "./implementations/SensitivitySetting";
+
+import type { AvailableValuesType } from "../interfacesAndTypes/utils";
+
+export enum SettingCategory {
+    SINGLE_OPTION = "singleOption",
+    MULTI_OPTION = "multiOption",
```

Comment:
Consider renaming to `SINGLE_SELECT` and `MULTI_SELECT` as both setting categories has multiple options, but differentiate in how many options can be select simultaneously.